src/Core__Null.res

```diff
@@ -34,6 +34,12 @@ let getExn: t<'a> => 'a = value =>
 
 external getUnsafe: t<'a> => 'a = "%identity"
 
+let forEach = (value, f) =>
+  switch value->toOption {
+  | Some(x) => f(x)
+  | None => ()
+  }
```

Comment:
Maybe it'd be more idiomatic now to do a pattern match on the actual null value? Instead of converting it to option.

Reply:
I agree, that was my first thought, but this would require rescript 11 and would not be compatible with previous versions. 

I would be in favor of requiring v11 for Core anyway. 

We could then rewrite pretty much every function in those modules.

Reply:
Of course 🤦 Let's go for this for now. We're going to go thorough all APIs thoroughly to align them with V11 (and uncurried) before we merge it into the compiler anyway.